indico/modules/events/timetable/templates/pdf/timetable.html

```diff
@@ -0,0 +1,79 @@
+{% from 'events/timetable/pdf/_cover_page.html' import render_cover_page %}
+{% from 'events/timetable/pdf/_toc.html' import render_toc %}
+{% from 'events/timetable/pdf/_program.html' import render_program %}
+
+<style>
+    @page {
+        size: {{ page_size }};
+        margin: 10mm;
+    }
+
+    @page :first {
+        counter-reset: toc-page;
+        counter-reset: program-page 0;
+
+        @bottom-center {
+            content: '{{ now|format_date(format='full', timezone=event.tz) }}';
+        }
+    }
+
+    @page toc {
+        counter-increment: toc-page;
+
+        @bottom-center {
+            content: counter(toc-page, lower-roman);
+        }
+    }
+
+    {% for day in days %}
+        @page program-{{ loop.index }} {
+            counter-increment: program-page;
+
+            @bottom-center {
+                content: counter(program-page);
+            }
+
+            @top-left {
+                font-size: 0.8em;
+                color: grey;
+                content: '{{ event.title }} / Programme';
```

Comment:
What happens if the event title contains a `'`, or a linebreak (some very legacy events have that... but we could just get rid of that nonsense in any case)?

Maybe safer to reference an attributes from here? `content: attr(data-whatever)` and then putting the actual text in a data attribute in the HTML code.

Reply:
Actually, not possible as we are not referencing an HTML element but `@top-left`. Will manually escape the string.